core/executor/src/fixed_types/mod.rs

```diff
@@ -54,67 +48,13 @@ impl FixedAsset {
 
 impl ContractSer for FixedAsset {
```

Comment:
The `ContractSer` is a useless trait,  any struct that implements the `Fixed-codec` can be stored in the `state trie`.

Reply:
> The `ContractSer` is a useless trait, any struct that implements the `Fixed-codec` can be stored in the `state trie`.

What if there are contract-specific structs  not implement in protocol/types?
e.g., structs in crosschain contract